cmd/kubeadm/app/cmd/init.go

```diff
@@ -19,27 +19,26 @@ package cmd
 import (
 	"fmt"
 	"io"
-	"io/ioutil"
 	"path"
 	"strconv"
 	"text/template"
 
 	"github.com/renstrom/dedent"
 	"github.com/spf13/cobra"
 
-	"k8s.io/apimachinery/pkg/runtime"
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/validation"
+	certphase "k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases"
```

Comment:
`certphase` would make sense for `k8s.io/kubernetes/cmd/kubeadm/app/phases/certs`, but not for `.../app/cmd/phases` which contain the sub-commands for the phases.